frontend/src/views/FormContainer.tsx

```diff
@@ -10,28 +10,44 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-import { Banner } from 'core/banner/Banner';
-import { Text } from 'core/text/Text';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
 import PropTypes from 'prop-types';
 import React from 'react';
 import styles from './FormContainer.module.css';
 
 const propTypes = {
   title: PropTypes.string.isRequired,
   subtitle: PropTypes.string.isRequired,
-  banner: PropTypes.node,
   children: PropTypes.node.isRequired,
 };
 
-export const FormContainer = ({ title, subtitle, banner, children }) => {
-  let bannerContent = banner ? <Banner data-testid="banner" content={banner} /> : null;
+const useUploadProjectViewStyles = makeStyles((theme) => ({
+  title: {
+    fontSize: '32px',
+    fontWeight: 'bold',
+    color: '#261e58',
+  },
+
+  subtitle: {
+    fontSize: '14px',
+    color: '#4b3baf',
+  },
+}));
+
+export const FormContainer = ({ title, subtitle, children }) => {
+  const classes = useUploadProjectViewStyles();
+
   return (
     <div className={styles.container}>
       <div className={styles.titleContainer}>
-        <Text className={styles.title}>{title}</Text>
-        <Text className={styles.subtitle}>{subtitle}</Text>
+        <Typography variant="h1" className={classes.title}>
```

Comment:
Do not rely on custom CSS to change that, use a proper variant instead as defined in the MaterialUI documentation. Theme providers will define how the variant look. Use something like `h4` and `h5` here.

Reply:
done